tests/php/ORM/DataObjectDuplicationTest.php

```diff
@@ -48,8 +48,8 @@ public function testDuplicate()
         );
         $this->assertEmpty(
             array_intersect(
-                $orig->bobcats()->getIDList() ?? [],
-                $duplicate->bobcats()->getIDList()
+                $orig->bobcats()->sort(null)->column('ID') ?? [],
+                $duplicate->bobcats()->sort(null)->column('ID')
```

Comment:
```suggestion
                $orig->bobcats()->column('ID') ?? [],
                $duplicate->bobcats()->column('ID')
```
This test is at risk of sporadically failing

Reply:
`array_intersect` doesn't care about sort order but reverted anyway